examples/redis-newvpc/main.tf

```diff
@@ -40,11 +35,11 @@ module "vpc" {
   enable_dns_hostnames    = true
   map_public_ip_on_launch = false
 
-  tags = local.tags
 }
 
 module "elasticache_redis" {
-  source                     = "intel/aws-elasticache-redis/intel"
+  #source                     = "intel/aws-elasticache-redis/intel"
+  source                     = "../../"
```

Comment:
Update

Reply:
I fixed this and pushed back to duckandbunny.  Also this is where i was testing the auto tagging so the new autotagging is part of this module with the last push i just made.